js/ui/windowManager.js

```diff
@@ -2765,4 +3136,52 @@ var WindowManager = class {
             }
         }
     }
+
+    _windowCanWobble(window, op) {
```

Comment:
optional: It's better not to use `window` as a variable name because it shadows the global object named `window`. In the case where the inner variable changes then the linter won't flag places where its uses need to change. But if this worked fine already, feel free to ignore it.

Reply:
Has been changed to 'win'.